package.json

```diff
@@ -1,48 +0,0 @@
-{
-  "name": "@caerostris/parallel-transform-stream",
-  "version": "0.0.1",
-  "description": "Transform streams in parallel",
-  "main": "dist/parallel-transform.js",
-  "scripts": {
-    "build": "babel ./src --out-dir ./dist",
```

Comment:
Okay, just thought as it is ES5 it’ll work. But there were breaking changes. Fine then.